pandas/tests/frame/conftest.py

```diff
@@ -67,6 +67,18 @@ def datetime_frame():
     return DataFrame(tm.getTimeSeriesData())
 
 
+@pytest.fixture
+def date_range_frame():
+    """
+    Fixture for DataFrame of ints with date_range index
```

Comment:
not in love with this as a generalized fixture, if you put in test_asof would be ok

Reply:
So far, the requirement was always that this should be on `tests/frame/conftest` to be reusable for other test modules. But I don't care either way - can put it into `test_asof.py` as well.